src/config.js

```diff
@@ -213,6 +213,10 @@ export const widgets = {
     ...defaultWidgets.id,
     description: CharCounterDescriptionWidget,
     cookie_consent_configuration: MultilingualWidget(),
+    data_conclusione_incarico: (props) => (
+      <DatetimeWidget {...props} dateOnly="true" />
```

Comment:
```suggestion
      <DatetimeWidget {...props} dateOnly={true} />
```

Reply:
Questo è inutile, nel senso che il secondo è più compatto e fa la stessa roba

Reply:
Tieni ragione. Il modo che preferisco è quello in cui passo il true... Explicit is better than implicit dicono le PEP